Quesnelia/app-consortia.json

```
@@ -12,17 +12,17 @@
   ],
   "modules": [
     {
-      "id": "mod-consortia-keycloak-1.4.0",
+      "id": "mod-consortia-keycloak-1.4.1",
       "name": "mod-consortia-keycloak",
-      "version": "1.4.0",
-      "url": "https://mdr-folio-eis-us-east-1-dev.s3.amazonaws.com/mod-consortia-keycloak-1.4.0"
+      "version": "1.4.1",
+      "url": "https://mdr-folio-eis-us-east-1-dev.s3.amazonaws.com/mod-consortia-keycloak-1.4.1"
     }
   ],
   "uiModules": [
     {
-      "id": "folio_consortia-settings-1.0.2",
+      "id": "folio_consortia-settings-1.1.0",
       "name": "folio_consortia-settings",
-      "version": "1.0.2",
+      "version": "1.1.0",
       "url": "https://folio-registry.dev.folio.org/_/proxy/modules/folio_consortia-settings-1.0.2"
```

```suggestion
      "url": "https://folio-registry.dev.folio.org/_/proxy/modules/folio_consortia-settings-1.1.0"
```
MD version not changed